### PR TITLE
Eliminate the “not-supported” option for tokenBinding.status

### LINKED ISSUE #907: Meaning of missing tokenBinding in ClientCollectedData?
the tokenBinding element is not required in ClientCollectedData. Is this meant to be a required value, or is perhaps the meaning when missing equivalent to "not-supported" or some other implicit status "unspecified"?

### PR DESCRIPTION
**Pull request by agl**
The tokenBinding member is optional so this created two different ways
to encode that tokenBinding wasn't supported: omitting tokenBinding
completely and including it with status = “not-supported”. This change
eliminates the second option.

This matches Firefox's current behaviour and Chrome will align.

Fixes #907.

***
<a href="https://pr-preview.s3.amazonaws.com/agl/webauthn/pull/914.html" title="Last updated on Jun 19, 2018, 4:17 PM GMT (7ef6a96)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/914/4fd5dd5...agl:7ef6a96.html" title="Last updated on Jun 19, 2018, 4:17 PM GMT (7ef6a96)">Diff</a>

### CODE REVIEW

**Comment by agl** [comment]
Good point about being clear on the meaning of omitting `tokenBinding`. I've updated the PR to cover that.

**Comment by emlun** [comment]
This will conflict with #901.

**Comment by equalsJeffH** [comment]
merge after #901 per @agl comment on call today

File: index.bs [hunk 1]
```diff
@@ -2007,17 +2007,14 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
         the syntax defined by [[RFC6454]].
 
     :   <dfn>tokenBinding</dfn>
-    ::  This member contains information about the state of the [=Token Binding=] protocol used when communicating with the
-        [=[RP]=].
+    ::  This optional member contains information about the state of the [=Token Binding=] protocol used when communicating
```
Comment (emlun) [inline index.bs]: Please change "optional" to "OPTIONAL" here.

Comment (agl) [reply index.bs]: Done.

**Comment by equalsJeffH** [comment]
merge this?

**Comment by nadalin** [comment]
@agl Please merge

**Pull request merged**